src/measure_theory/bochner_integration.lean

```diff
@@ -51,7 +51,7 @@ The Bochner integral is defined following these steps:
 2. Basic properties of the Bochner integral on functions of type `α → ℝ`, where `α` is a measure
   space.
 
-  * `integral_nonneg_of_nonneg_ae`  : `∀ₘ a, 0 ≤ f a → 0 ≤ ∫ f`
+  * `integral_nonneg_of_ae`  : `∀ₘ a, 0 ≤ f a → 0 ≤ ∫ f`
```

Comment:
could you make sure that the colon is aligned with the one on the following line?